pages/Eye.md

```diff
@@ -7,9 +7,9 @@
 In dieser Aufgabe sollen Sie ein Auge zeichnen, dessen Pupille der Maus folgt.
 Das heißt, das Auge soll immer an der gleichen Stelle stehen, die Pupille soll aber in Richtung der Maus schauen.
 Zeichnen Sie zuerst einen Kreis in der Mitte der Zeichenfläche, der das Auge darstellt.
-Wenn sich die Pupille innerhalb des Auges befindet, soll die Pupille dort gezeichnet werden, wo der Mauszeiger ist.
+Wenn sich die Maus innerhalb des Auges befindet, soll die Pupille dort gezeichnet werden, wo der Mauszeiger ist.
```

Comment:
Ich hatte die von dir vorgeschlagene Formulierung explizit nicht gewählt. Das Problem ist, dass der Mauszeiger sich innerhalb des Auges aber zu weit am Rand befinden kann, so dass die Pupille nicht mehr im Auge wäre. Ich denke auch, dass die aktuelle Formulierung nicht gut ist. Die Formulierung mit der Maus ist vermutlich besser. Meinst du, man muss noch klarstellen, dass die Bedingung so nicht ganz richtig ist?

Reply:
Ich hing an der Stelle auch ein paar Minuten - ich weiß auch genau was du meinst, beide Formulierungen sind hier nicht ideal. Wahrscheinlich sollte der Abschnitt umformuliert werden - in etwa so: 

Die Pupille folgt dem Mauszeiger - solange sie dabei im Auge bleibt, wird sie direkt am Mauszeiger gezeichnet, ansonsten wird sie am Rand des Auges gezeichnet, die Richtung vom Augenmittelpunkt zur Pupille ist dabei die gleiche wie vom Augenmittelpunkt zum Mauszeiger.

Ich meine, vielleicht braucht man für diese Aufgabe sogar noch weniger Erklärung. Für die meisten sollte eigentlich verständlich sein, was es bedeutet, dass `ein Auge an fester Position immer auf den Mauszeiger schauen soll`.